packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton-container {
```

Comment:
I think this code should go into the docs project; this file should _only_ expose `.pt-skeleton-bone` as a public API.